OpenOversight/app/templates/about.html

```diff
@@ -1,150 +1,158 @@
-  </div>
-</div>
 
-<div class="pb-50"><div class="hr-banner">
-    <h2>Press Release</h2>
-    <p class="text-left">In support of demands for greater police accountability, Illinois nonprofit The Lucy Parsons Labs launched
-      OpenOversight, an interactive web tool and accountability platform that makes it easier for the public to identify
-      police officers, including for the purpose of complains. We rely on crowdsourced and public data to build a
-      database
-      of police officers in a city, allowing the public to filter through the dataset to find the name and badge number
-      of
-      the offending officer.</p>
-
-    <p class="text-left">Using OpenOversight, members of the public can search for the names and badge numbers of police with whom they
-      have negative interactions using the officer's estimated age, race and gender. Using this information, the
-      OpenOversight web application returns a digital gallery of potential matches and, when possible, includes pictures
-      of officers in uniform to assist in identification. "The deck is stacked against people harmed by police," says
-      Jennifer Helsby, CTO of the Lucy Parsons Labs and lead developer on the OpenOversight project. "Police are almost
-      never held accountable for misconduct or crimes they commit. To file a misconduct complaint, the burden is on the
-      public to provide as much detailed data about the officer as possible. OpenOversight aims to empower citizens with
-      tools that make it easier to identify officers and hold them accountable."
-    </p>
-  </div>
-</div>
-
-<div class="horizontal-padding vertical-padding">
-  <div class="text-center">
-    <h2>Facts and Figures about the Chicago Police Department</h2>
+    <div class="row font-weight-300 text-center display-flex">
+      <div class="col-lg-4 col-half-border">
+        <span class="fa fa-institution fa-2x"></span>
+        <p>
+          This project is a response to the lack of transparency and justice in
+          policing. The public should have the right to know which officers are
+          patrolling their neighborhoods and watching their communities. When
+          officers abuse their positions of power, they should be able to be
+          easily identified and held accountable.
+        </p>
+      </div>
+      <div class="col-lg-4 col-half-border">
+        <span class="fa fa-map-marker fa-2x"></span>
+        <p>
+          It is the first project of its kind in the United States, and was
+          first implemented in Chicago in October 2016. OpenOversight launched
+          in the East Bay of the San Francisco Bay Area in fall 2017 and in New
+          York City in 2018. A Baltimore instance was launched in 2019 at
+          <a
+            href="https://bpdwatch.com"
+            rel="noopener noreferrer"
+            target="_blank"
+            >BPDWatch.com</a
+          >.
```

Comment:
We could probably mention the Portland instance (cops.photo) launched around the same time too

Reply:
Good idea! Done in 1a94762